src/app-layer-dnp3.h

```diff
@@ -15,6 +15,12 @@
  * 02110-1301, USA.
  */
 
+/**
+ * \file
+ *
+ * DNP3 application layer protocol header file
```

Comment:
So i should remove the header tag?

If so...okay.

Reply:
I think you can leave just as I suggested - I'm saying that just because it was pointed out that indicating that this was a header file was redundant.

Reply:
Oh sure.